backend/kernelCI_app/views/treeDetailsSlowView.py

```diff
@@ -155,14 +155,17 @@ def __getCurrentRowData(self, currentRow):
         testError = extract_error_message(currentRow[tempColumnDict["tests_misc"]])
         testEnvironmentCompatible = currentRow[
             tempColumnDict["tests_environment_compatible"]
-        ]
+        ][0] if currentRow[
+            tempColumnDict["tests_environment_compatible"]
+        ] is not None else None
 
         historyItem = {
             "id": testId,
             "status": testStatus,
             "path": path,
             "duration": testDuration,
             "startTime": currentRow[tempColumnDict["tests_start_time"]],
+            "hardware": currentRow[tempColumnDict["tests_environment_compatible"]]
```

Comment:
ditto about reuse variable (`testEnvironmentCompatible`)

Reply:
testEnvironmentCompatible gets only the first element and here I want them all